### PR TITLE
publish-commit-bottles: Improve the style of "merge triggered" message

### PR DESCRIPTION
**Pull request by issyl0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] ~Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?~
- [x] ~Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?~
- [x] ~Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?~

-----

- It was probably not entirely clear to users what "bottle pull" meant.
- We can use Markdown style links to make the message shorter.
- Tested in a browser with `console.log()` as that's as far as my JS goes!

### CODE REVIEW

File: .github/workflows/publish-commit-bottles.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
             github.issues.createComment({
               ...context.repo,
               issue_number: pr,
-              body: '@' + actor + ' has triggered a merge, running bottle pull: ' + url
+              body: '@' + actor + ' has triggered a merge. [View the progress logs](%s).', url
```
Comment (issyl0) [inline .github/workflows/publish-commit-bottles.yml:47]: At the risk of bikeshedding, how about:

```suggestion
              body: '@' + actor + ' has triggered a merge. [View its progress](%s).', url
```

Comment (issyl0) [reply .github/workflows/publish-commit-bottles.yml:47]:
```suggestion
              body: '@' + actor + ' has triggered a merge. [View the merge logs](%s).', url
```

❓

Comment (issyl0) [reply .github/workflows/publish-commit-bottles.yml:47]:
```suggestion
              body: '@' + actor + ' has [triggered a merge](%s).', url
```

Comment (Bo98) [reply .github/workflows/publish-commit-bottles.yml:47]: Yeah I was thinking about the last one. Generally, the less words the better.

**Comment by issyl0** [comment]
Thanks, @Bo98 - I had decision paralysis and wasn't entirely sure I could accurately word English for a minute there!

I've done the "least words" thing for both the cases where it'll post a message, for consistency.

**Pull request merged**